src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Deprecate calling `Button/Form::setParent()`, `ButtonBuilder/FormConfigBuilder::setDataMapper()`, `TransformationFailedException::setInvalidMessage()` without arguments
  * Change the signature of `FormConfigBuilderInterface::setDataMapper()` to `setDataMapper(?DataMapperInterface)`
  * Change the signature of `FormInterface::setParent()` to `setParent(?self)`
+ * Add `hash_property_path` option to `PasswordType`
```

Comment:
```suggestion
 * Add `PasswordHasherExtension` with support for `hash_property_path` option in `PasswordType`
```

Or something like this? (the extension must be enabled explicitly when using the component standalone)